### ISSUE TITLE
crash when talking to DoA snake

### ISSUE THREAD

**Issue by HasKha** [HasKha/GWToolboxpp#620]
Crash happens when clicking on the snake at the start of foundry room 5. I believe it happens when the UI window is created, it has nothing to do with taking quests. It might be happening in other places too, not sure. 

Crashed in (part of call stack):
- `StoCHandler_Func`, packet 182
- UIMgr.cpp, line 139, on the RetSendUIMessage call (insider OnSendUIMessage)
- on string decode? not sure

I can send the dump file

**Comment by 3vcloud** [comment]
Dump would be good yep, the ui message handler is pretty solid so would be surprised if that was the cause, but the message I'd might be a hint if its in the dump. Did you have obfuscator on?

**Comment by HasKha** [comment]
I did not have obfuscator on, but I did see some messages on the console about guild obfuscating, not sure what's up about that. 

Note that it does not happen consistently. It was suggested that I click on the snake before the door opens. The npc updates when the door opens, so maybe that has an effect. 

I remember seeing the string "Accept" in the variables when the crash happened. Unfortunately I had to close that quickly. I saved the dump but I can't restore from there, I'm doing something wrong. 

https://1drv.ms/u/s!AicDyQVrn8hX2EiDCVvJJV0FXtrP?e=fCCGuG
